core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -247,6 +250,7 @@ oppia.controller('FeedbackTab', [
               var suggestion = $scope.activeThread;
               var stateName;
               if (constants.USE_NEW_SUGGESTION_FRAMEWORK) {
+                suggestion = $scope.activeThread.suggestion;
```

Comment:
Do an actual if/else here. Don't set it on lines 250/251 and then overwrite it to something else. The only value that's fine to use on lines 250/251 as a "default" is null.

Reply:
Done